compiler/src/dotty/tools/repl/Rendering.scala

```diff
@@ -33,7 +33,8 @@ private[repl] class Rendering(parentClassLoader: Option[ClassLoader] = None) {
 
   /** A `MessageRenderer` for the REPL without file positions */
   private val messageRenderer = new MessageRendering {
-    override def posStr(pos: SourcePosition, diagnosticLevel: String, message: Message)(using Context): String = ""
+    override def posStr(pos: SourcePosition, diagnosticLevel: String, message: Message)(using Context): String =
+      s"-- $diagnosticLevel:"
```

Comment:
might be nice just to color the warning/error header, tests shouldnt need updating:
```suggestion
      hl(diagnosticLevel)(s"-- $diagnosticLevel:")
```